kubecost-cloud/using-the-kubecost-cloud-ui/kubecost-cloud-request-right-sizing.md

```diff
@@ -1,7 +1,7 @@
 # Kubecost Cloud: Request Right-Sizing Recommendations
 
 {% hint style="info" %}
-This documentation should only be consulted when using Kubecost Cloud! For information about request right-sizing recommendations for self-hosted Kubecost, see [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/auto-request-sizing).
+This documentation should only be consulted when using Kubecost Cloud! For information about request right-sizing recommendations for self-hosted Kubecost, see [here](/using-kubecost/navigating-the-kubecost-ui/savings/container-request-right-sizing-recommendations.md).

```

Comment:
More of a meta-comment: changing doc slugs breaks bookmarks that users may have saved, have you thought about a good way of handling that? Maybe gitbook does it already

Reply:
I can see the value of it, but have not looked into how it could be implemented yet.